MIT.md

```diff
@@ -1,10 +1,23 @@
 ## Copyright 2021, [Sana Tameem]
 
-###### Please delete this line and the next one
-###### APP TYPE can be a webpage/website, a web app, a software and so on
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software application and associated documentation files (the "OOP School Library"), to deal in the OOP School Library without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the OOP School Library, and to permit persons to whom the OOP School Library is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the OOP School Library.
+
+THE OOP SCHOOL LIBRARY IS PROVIDED "AS IS," WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES, OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT, OR OTHERWISE, ARISING FROM, OUT OF, OR IN CONNECTION WITH THE OOP SCHOOL LIBRARY OR THE USE OR OTHER DEALINGS IN THE OOP SCHOOL LIBRARY.
+
```

Comment:
- [ ] [REQUIRED] As recommended by the previous reviewer. Kindly customize your license properly. I can see you already implemented this but it is not well linked in your README. You only resolved the issue on the MIT file but the URL of the MIT attached to the README is wrong.

Take a look at the wrong URL you attached:
https://github.com/SanaTameem/OOP--school-library/blob/6b32035a164e5242d50f9b022c4f53ccdf3f3f77/README.md?plain=1#L139
This URL is from your `add/classes` branch.

To resolve this issue, use this URL:
This project is [MIT](MIT.md) licensed.

Reply:
Thanks for reviewing the project.
But when I opened the feature/association branch the MIT file was correct. 
The picture that you provided is from my previous project's branch(feature/add-classes)
Currently, I made a PR from the feature/association to the dev branch.
![Capture](https://github.com/SanaTameem/OOP--school-library/assets/71990110/f3ab9f5d-ad72-4bdc-91e1-19f37122bdbf)

Reply:
Sir, let me know if still I need to change the MIT file.
Should I request a review without changing or still some problems exist there?